fsspec/implementations/tests/test_local.py

```diff
@@ -939,32 +939,24 @@ def test_cp_get_put_empty_directory(tmpdir, funcname):
     # cp/get/put without slash, target directory exists
     assert fs.isdir(target)
     func(empty, target)
-    assert fs.find(target, withdirs=True) == [
-        make_path_posix(os.path.join(target, "empty"))
-    ]
-
-    fs.rm(target + "/empty", recursive=True)
+    assert fs.find(target, withdirs=True) == []
```

Comment:
This test is effectively duplicated in the abstract test harness, so it can eventually be removed. But I have fixed it here for the time being.